syntax = "proto3";

service Greeter {
  rpc SayHello (HelloRequest) returns (HelloReply) {}
}

message HelloRequest {
  string name = 1;
  int32 age = 2;
}

message HelloReply {
  string greetings = 1;
}

service Calculator {
  rpc Sum (SumRequest) returns (SumResponse) {}
}

message SumRequest {
  int32 a = 1;
  int32 b = 2;
}

message SumResponse {
  int32 c = 1;
}
